src/components/node/stanford-page/page-display.tsx

```diff
@@ -14,9 +17,28 @@ const StanfordPage = async ({node, ...props}: {node: NodeStanfordPage}) => {
 
   return (
     <div {...props}>
-      <StanfordPageMetadata node={node} />
-      {node.suPageComponents && <Rows components={node.suPageComponents} fullWidth={fullWidth} />}
-      <footer className="rs-py-4 centered">Last updated {lastUpdated}</footer>
+      <StanfordPageMetadata node={node} />{" "}
+      <InternalHeaderBanner>
+        <h1
+          id={node.id}
+          className="relative mx-auto mb-10 mt-75 flex w-full max-w-[calc(100vw-10rem)] flex-row gap-20 p-0 md:max-w-[calc(100vw-20rem)] 3xl:max-w-[calc(1500px-20rem)]"
+        >
+          <RosetteIcon width={60} height={60} />
+          {node.title}
+        </h1>
+      </InternalHeaderBanner>
+      {!fullWidth && (
+        <InteriorPage node={node} currentPath={node.path || "#"}>
+          {node.suPageComponents && <Rows components={node.suPageComponents} fullWidth={fullWidth} />}
+          <footer className="rs-py-4 centered">Last updated {lastUpdated}</footer>
+        </InteriorPage>
+      )}
+      {fullWidth && (
+        <>
+          {node.suPageComponents && <Rows components={node.suPageComponents} fullWidth={fullWidth} />}
+          <footer className="rs-py-4 centered">Last updated {lastUpdated}</footer>
```

Comment:
Since the footer is the same in both fullwidth and not fullwidth, can we just pull it out of the conditionals. Then we can remove the fragment.